src/ensembl/src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewState.ts

```diff
@@ -0,0 +1,55 @@
+export type EntityViewerGeneFunctionState = {
+  selectedTabName: GeneFunctionTabName;
+};
+
+export type EntityViewerGeneRelationshipsState = {
+  selectedTabName: GeneRelationshipsTabName;
+};
+
+export type EntityViewerGeneViewObjectState = {
```

Comment:
`GeneViewObject` doesn’t tell us much

`EntityViewerGeneState`?
or perhaps `EntityViewerGeneUIState`? or `EntityViewerGeneViewUIState`?

Reply:
Changed it to `EntityViewerGeneViewUIState` 👍